### DISCUSSION TITLE
Emulating ASUS FanXpert/AI Suite 'Extreme Quiet' settings

### DISCUSSION THREAD

**Discussion by PoizenJam** [Rem0o/FanControl.Releases#1782] [Ideas]
Not sure if this is an outright feature request, but it doesn't feel like an issue and I may just be missing something, so I'm posting it as an 'idea'

Modern ASUS FanXpert software allows the user to enable an 'Extreme Quiet' mode. This allows fans to run below their stated specs by a significant margin. For example, my Lian Li ST120 fans that are rated for 800-1900RPM officially, with 800RPM corresponding to ~40% fan speed. By enabling 'Extreme Quiet' mode, you can extend the viable fan curve range down to ~22-25%, allowing you to drop the fan speed to at low as 450RPM.

I am unsure how ASUS accomplishes this, but I cannot replicate these extremely low speeds in Fan Control. Seemingly even if I modify the Start/Stop % or minimum speeds. I'm lead to assume ASUS utilizes some proprietary 'switch' behind the scenes to make this happen- perhaps halving the voltage or RPM signal. The reason is that, if I turn Fan Control and override FanXpert _while 'Extreme Quiet' is active_, Fan Control will be _stuck_ in 'Extreme Quiet' mode. That is, min and max speed in Fan Control will now be mapped to 450RPM and 800RPM, respectively. You cannot exceed 800RPM without re-activating ASUS FanXpert, disabling 'Extreme Quiet', then going back to Fan Control.

Would I be correct to assume ASUS 'Extreme Quiet' is beyond the scope of anything Fan Control was intended to do? And are any workarounds possible?

**Comment by Rem0o** [comment]
Remapping the power range, or whatever Asus is doing behind the scenes, is a Asus specific thing. Fan Control aims to stay generic in its functionalities, so yes, out of scope.

**Reply by PoizenJam** [reply]
Super thanks for responding, and for all you do with Fan Control. I wasn't sure how ASUS was able to achieve these sub-spec'd RPMs for different fans, but I suspected it might be some proprietary motherboard shenanigans. 

Unfortunate, but totally understandable that this feature is outside the scope of Fan Control :)
